scripts/deploy-container-image.sh

```diff
@@ -19,14 +19,12 @@ login() {
 
 
 deploy_docker() {
-    local tag=$1
-    [[ "$tag" =~ ibm-openapi-validator@([[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+) ]]
-    local docker_tag=${BASH_REMATCH[1]}
```

Comment:
It was easier to give the script the raw version (`x.x.x`), which also saved us the parsing step, so I went with that.

Also, I decided to rename the file to clarify what exactly was being deployed